### ISSUE TITLE
soft blurry images from 2.0.0 update

### ISSUE THREAD

**Issue by Marscog** [MoonRide303/Fooocus-MRE#79]
The previous version 1.0.64 MRE was giving me prefect images. Everything I've done in the 2.0.0 update is slightly soft and blurry and the faces are not good. It actually bothers my eyes looking at the new images, because my eyes keep trying to focus the images but I can't because they are slightly out of focus just enough to be irritating.
Here is an example. I used load prompt to render the image in 2.0.0. The old version 1.0.64 only has one woman and the image is great. The 2.0.0 image has two women and is soft with bad faces.
Right now, I'll just keep using version 1.0.64 MRE, because it is generating the best images I've ever seen so far.
Maybe I have some setting set wrong, but I can't find it.
![2023-09-11_12-07-34_8671](https://github.com/MoonRide303/Fooocus-MRE/assets/142053317/76897af9-0e8f-4cdb-b80d-98c8415974ee)
![2023-09-13_00-23-16_4662](https://github.com/MoonRide303/Fooocus-MRE/assets/142053317/af5e7ae2-a94b-4b48-b509-bb1c65c72777)

**Comment by MoonRide303** [comment]
@Marscog Seems to be the same issue as #75 - the difference like that were caused by float autocasting in Comfy / CLIP not working properly in CLIP for the last 3 weeks.

Vanilla Fooocus uses earlier Comfy version with autocast working, in MRE I sticked to Comfy from master, so last 3 weeks I had the same problem as it was observed in pure ComfyUI. I talked with comfy about it, and from his perspective it was a bug and inconsistent behaviour within CLIP between float operations. He doesn't want to make old behaviour available even by manually using some parameters, as he considered it a bug.

So if you're happy with this particular output produced by bugged comfy version I'd suggest just sticking to v1.0.64 for a moment. I've added tag for this version, so it's easier to download from git for people who want it this way.
